packages/components/notification/__tests__/notification.test.tsx

```diff
@@ -129,6 +118,7 @@ describe('Notification.vue', () => {
       const type = 'some-type'
       const wrapper = _mount({
         props: {
+          // @ts-ignore
```

Comment:
Is there any other way to solve this problem? 🤔

Reply:
I asked for the opinion of **sxzz**, it would be more reasonable to use `@ts-expect-error`.